XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,602 @@
+
+## 7. Transaction: `CredentialDelete`
+
+This transaction deletes a `Credential` object.
+
+It can be executed by:
+* The issuer, anytime.
+* The account, anytime.
+* Anyone, after the expiration time is up.
+
+Deleting a credential is also how a credential is un-accepted.
+
+### 7.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`TransactionType`| ✔️|`string`|`UInt16`|The transaction type (`CredentialDelete`).|
+|`Account`| ✔️|`string`|`AccountID`|The transaction submitter.|
+|`Subject`| |`string`|`AccountID`|The person that the credential is for. If omitted, `Account` is assumed to be the subject.|
+|`Issuer`| |`string`|`AccountID`|The issuer of the credential. If omitted, `Account` is assumed to be the issuer.|
+|`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+
+_Note: If an account is deleting a credential it issued to itself, then either `Subject` or `Issuer` can be specified, but at least one must be._
+
+### 7.2. Failure Conditions
+
+* The credential described by the `Subject`, `Issuer`, and `CredentialType` fields doesn't exist.
+* The `Account` isn't the issuer or subject, and the expiration hasn't passed.
+
+### 7.3. State Changes
+
+If the transaction is successful:
+* The `Credential` object is deleted.
+
+## 8. Any Transaction Affected by Deposit Authorization
+
+### 8.1. Fields
+
+As a reference, [here](https://xrpl.org/docs/references/protocol/transactions/common-fields/) are the fields that all transactions currently have.
+
+The transactions that this field will be added to are:
+* `Payment`
+* `EscrowFinish`
+* `PaymentChannelClaim`
+* `AccountDelete`
+
+| Field Name | Required? | JSON Type | Internal Type |
+|------------|-----------|-----------|---------------|
+|`CredentialIDs`| |`array`|`Vector256`|Credential(s) to attach to the transaction.|
+
+### 8.2. Failure Conditions
+
+* Any of the `CredentialIDs`:
+  * Isn't an object that exists.
+  * Isn't a `Credential` object.
+  * Is an expired `Credential` object.
+  * Has not been accepted.
+  * Isn't a credential issued to the `Account` sending the transaction.
+* `CredentialIDs` is included, but the destination doesn't have Deposit Authorization set up, or the transaction doesn't have a destination (e.g. `AccountSet`).
+* The group of `CredentialIDs` is not authorized by the destination.
+
+Note: the transaction will still fail if too many credentials are included. The exact list must be provided.
+
+### 8.3. State Changes
+
+* If the credential isn't valid, the transaction fails.
+* If the credential is expired, the credential is deleted.
+
+## 9. RPC: `deposit_authorized`
+
+The [`deposit_authorized` RPC method](https://xrpl.org/deposit_authorized.html) already exists on the XRPL. This proposal suggests some modifications to also support credential authorization.
```

Comment:
old link

https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/deposit_authorized